### PR TITLE
add brazil to readme

### LINKED ISSUE #1342: Add Brazil to readme.txt

### PR DESCRIPTION
**Pull request by leonardola**
Fixes #1342

## Changes proposed in this Pull Request:
This PR adds brazil to the accepted countries. 
Also it updates the readme to Markdown. (But I am ok to revert it to .txt if it is necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,71 +63,71 @@ Stripe is available for Store Owners and Merchants in:
...
 1. The Stripe payment gateway settings screen used to configure the main Stripe gateway.
 2. Offer a range of payment methods such as local and alternative payment methods.
 3. Pay with a saved payment method, a new card, and allow customers to save the payment card for future transactions.
 4. Apple Pay and other Payment Request buttons can be used on the Product Page and Checkout for express checkout.
 
-== Changelog ==
+## Changelog ##
 
-= 5.x.x - 2021-xx-xx =
+# 5.x.x - 2021-xx-xx #
```
Comment (malithsen) [inline README.md]: We should probably use a smaller heading here
Screen Shot 2021-11-01 at 5 37 33 PM

**Review by malithsen — COMMENTED** [review]
This looks good to me @leonardola 
The version number field doesn't seem to be updated properly when it was converted to md. Let me know if this is intentional

**Comment by leonardola** [comment]
@malithsen can you please review again to accept the changes. I can't merge without at least one approval :/

**Review by malithsen — APPROVED** [review]
Looks good now, thanks for the update

**Pull request merged**